cmd/build.rs

```diff
@@ -1,8 +1,51 @@
 // Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
 
+fn link_cpp(tool: &cc::Tool) {
+    let stdlib = if tool.is_like_gnu() {
+        "libstdc++.a"
+    } else if tool.is_like_clang() {
+        "libc++.a"
```

Comment:
I can't find "libc++.a" with `clang++`, found "libstdc++.a" though.

Reply:
I believe TiKV cannot be built with clang++ yet. There are quite a lot errors when compiling RocksDB with clang

Reply:
Interesting, seems titan can't pass compilation
```
/git/checkouts/rust-rocksdb-a9a28e74c6ead8ef/1494bb5/librocksdb_sys/libtitan_sys/titan/src/db_impl.cc:105:14: error: variable 'file_size' set but not used [-Werror,-Wunused-but-set-variable]
      uint64_t file_size = 0;
               ^
  1 error generated.

```